packages/app/src/views/Root.tsx

```diff
@@ -54,7 +54,7 @@ function Root() {
     <div style={{backgroundColor:"white"}}>
         <Outlet />
     </div>
-    <StyledFooter>Group 2</StyledFooter>
+    <StyledFooter>© 2023 Group 2, Inc.</StyledFooter>
```

Comment:
Why hard code? The year could be programmatically created so this stays up to date?

Reply:
Inc 🔥 🚢

Reply:
{new Date().getFullYear()}

Reply:
const year = new Date().getFullYear()